### ISSUE TITLE
Puppet mirror isn't updating

### ISSUE THREAD

**Issue by penguian** [ACCESS-NRI/accessdev-Trac-archive#107]
|    by cma900@nci.org.au

___

Cron job fails to populate a newly created mirror.  Issue reported upstream:

https://github.com/ScottWales/puppet-git/issues/1

___
_Issue migrated from [trac:107](https://accessdev.nci.org.au/trac/107) at 2024-01-31 18:04:04 +1100_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
Appears to just be a timeout, manually doing a `git fetch origin`, `git push mirror` on /scratch/puppet.git-fetcher has worked. Will see if it continues to be updated.

**Comment by penguian** [comment]
_@chris.allen@anu.edu.au_ _commented_


___
That will work because you've explicitly specified which remote to fetch (cf. original report).  Once initial manual fetch/push is done, it will work after that but it shouldn't be necessary (hence low priority).

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
Ok, thanks for the info

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _removed milestone (was `Future`)_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
Milestone Future deleted
